packages/react-provider/src/ws.ts

```diff
@@ -29,9 +40,15 @@ export class WS {
 
     this.connection.onopen = this.onOpen.bind(this);
     this.connection.onclose = this.onClose.bind(this);
+    this.connection.onerror = this.onError.bind(this);
     this.connection.onmessage = this.onMessage.bind(this);
   }
 
+  onError(): void {
+    console.error("Error Connecting to Courier Push");
```

Comment:
can we get a more informative error here?

Reply:
lets wait for feedback and optimize premature